### DISCUSSION TITLE
Can I both output .jpg + .png?

### DISCUSSION THREAD

**Discussion by savage-weather** [mcmonkeyprojects/sd-infinity-grid-generator-script#83] [Q&A] [answered]
Thank you very much for making this amazing extension. I mainly use this extension to explore uncharted waters. I find it's very useful to find hidden gems that otherwise couldn't be found with limited axis settings.

One of the use cases where it might be beneficial to output both `jpg` and `png` is for online backup. I set up compression rate for `jpg` at 50 so it's much smaller than `png`, so the entire grid can be easily uploaded to github for safe keeping.

But `jpg` image quality low, I'd love to have the script saving both `jpg` and `png`. So if I want to upscale I just need to copy / move the `png` without having to re-generate the image.

**Comment by mcmonkey4eva** [comment, marked as answer]
This seems like something to do via external process tbh - generate it in `.png` (or high quality jpeg), and then have an external process that creates a copy of the folder with images converted to lowquality jpeg for backup.
